build-system/tasks/storybook-react/react-env/webpack.config.js

```diff
@@ -0,0 +1,75 @@
+const path = require('path');
+const {BUILD_CONSTANTS} = require('../../../compile/build-constants');
+const {DefinePlugin} = require('webpack');
+const {getRelativeAliasMap} = require('../../../babel-config/import-resolver');
+const {webpackConfigNoChunkTilde} = require('../../storybook/env-utils');
+
+const rootDir = path.join(__dirname, '../../../..');
+
+module.exports = ({config}) => {
+  config.resolveLoader = {
+    modules: [
+      path.join(__dirname, '../node_modules'),
+      path.join(rootDir, 'node_modules'),
+    ],
+  };
+  config.resolve = {
+    modules: [
+      path.join(__dirname, '../node_modules'),
+      path.join(rootDir, 'node_modules'),
+    ],
+    alias: {
+      // Alias the accordion
```

Comment:
Do we need to alias every extension's binaries manually? If so, not a huge fan since we're basically aliasing `**/extensions/*/1.0/component` -> `**/extensions/*/1.0/dist/component-react.max`. Is there a simpler way of doing this? Would it be simple to use a custom webpack-plugin? (Maybe this logic could be reused for running unit tests on react?)

Reply:
Weighing in a little: Aliases are tricky to get overly complicated with because many different systems (linter, VSCode auto-import/reference jumping, compiler, TypeScript, and webpack) all need to be on the same page, and some implement differently than others. Likely possible to do, but should probably be tackled as an isolated change given that it's no unlikely to introduce regressions in one or more of the areas and will be easier/safer to revert if that happens as an isolated change to import resolution ratherhan this whole PR

Reply:
I agree that all these aliases are pretty gross, but I couldn't find a better solution in a timely manner. We can try to investigate again later for something more elegant.

Reply:
To clarify, a better solution than what? Are the aliases needed to make it work for some reason?

Reply:
Yes, because the stories for storybook are importing the preact components directly, we need to map them to the react versions or else the react storybook won't load.

Reply:
Ahh gotcha. This seems like it could become increasingly unwieldy as the number of Bento components grows, but I don't see any clear alternatives without diving deep. Sounds like a job for another PR, thanks for entertaining my inquisition!